src/ontology/ro.Makefile

```diff
@@ -53,7 +53,7 @@ $(IMPORTDIR)/cob_import.owl: $(MIRRORDIR)/cob.owl $(IMPORTDIR)/cob_terms_combine
 	if [ $(IMP) = true ]; then $(ROBOT) query -i $< --update ../sparql/preprocess-module.ru \
     extract -T $(IMPORTDIR)/cob_terms_combined.txt --copy-ontology-annotations true --force true --method BOT \
 	remove $(patsubst %, --term %, $(ANNOTATION_PROPERTIES)) -T $(IMPORTDIR)/cob_terms_combined.txt --select complement \
-	remove --select "RO:* BFO:0000050* BFO:0000051* BFO:0000060* BFO:0000066*" --axioms logical \
+	remove --select "RO:* BFO:0000050* BFO:0000051* BFO:0000060* BFO:0000066*" --axioms "annotation logical" \
```

Comment:
I guess it is ok in this particular case, but note that my approach with --base-iri --select internal is safer as you do not delete _extensions_ (imagine a subclass axiom in the importing ontology, say :A subClassOf RO:001. This axiom would not be deleted with my approach, but with yours, which is mostly not right.

However here, I think we can make an exception as we do not expect COB to extend RO properties.

Reply:
Yes, I know. I tried the other one, and it didn't remove the logical axioms we needed to remove.

Reply:
I know why: https://github.com/ontodev/robot/pull/1187

Its ok lets roll with your solution now! Thanks!